akka-actor/src/main/scala/akka/serialization/Serialization.scala

```diff
@@ -246,10 +246,12 @@ class Serialization(val system: ExtendedActorSystem) extends Extension {
   val serializerByIdentity: Map[Int, Serializer] =
     Map(NullSerializer.identifier -> NullSerializer) ++ serializers map { case (_, v) ⇒ (v.identifier, v) }
 
-  private def shouldWarnAboutJavaSerializer(serializedClass: Class[_], serializer: Serializer) =
+  private def shouldWarnAboutJavaSerializer(serializedClass: Class[_], serializer: Serializer) = {
     settings.config.getBoolean("akka.actor.warn-about-java-serializer-usage") &&
       serializer.isInstanceOf[JavaSerializer] &&
```

Comment:
Could you please fix a small thing here, while you are at it. This method is only called once per class, but I'm still somewhat uneasy with performance implications of the config lookup. Please extract `settings.config.getBoolean("akka.actor.warn-about-java-serializer-usage")` to a `private val` in the `Serialization` class.